telegram/include/userver/telegram/bot/requests/request.hpp

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <userver/telegram/bot/utils/token.hpp>
+
+#include <userver/clients/http/request.hpp>
+
+USERVER_NAMESPACE_BEGIN
+
+namespace telegram::bot {
+
+/// todo
+struct RequestOptions {
+  /// todo
+  std::chrono::milliseconds timeout = std::chrono::milliseconds{500};
+
+  /// todo
+  size_t retries = 2;
+};
+
+template <typename method>
+class [[nodiscard]] Request {
```

Comment:
It looks like a real client class `client/client.hpp`, as it is actually should be.
_Anyway, `telegram/bot/client` is a better place for it._